lib/src/soloud_capture.dart

```diff
@@ -0,0 +1,245 @@
+import 'dart:async';
+import 'dart:ffi' as ffi;
+
+import 'package:flutter_soloud/src/enums.dart';
+import 'package:flutter_soloud/src/soloud.dart';
+import 'package:flutter_soloud/src/soloud_controller.dart';
+import 'package:logging/logging.dart';
+
+/// Use this class to _capture_ audio (such as from a microphone).
+///
+/// This class is completely independent from [SoLoud]. You can
+/// initialize and shut it down regardless of the state of [SoLoud].
+interface class SoLoudCapture {
+  /// A deprecated way to access the singleton [instance] of [SoLoudCapture].
+  ///
+  /// The reason this is deprecated is that it leads to code that misrepresents
+  /// what is actually happening. For example:
+  ///
+  /// ```dart
+  /// // BAD
+  /// var sound = await SoLoudCapture().loadFile('path/to/sound.mp3');
+  /// await SoLoudCapture().play(sound)
+  /// ```
+  ///
+  /// The code above suggests, on the face of it, that we're _constructing_
+  /// two instances of [SoLoudCapture], although we're in fact only accessing
+  /// the singleton instance.
+  @Deprecated('Use SoLoudCapture.instance instead')
```

Comment:
Since we have not yet published the plugin with `SoLoudCapture`, are these and the other deprecated advice necessary?

Reply:
This one isn't _but_ it help when migrating. For example, you have code like this:

```dart
SoLoud().initCapture();
```

You get a lint warning to change to `SoLoudCapture()`. You do that. Now you get another warning to change to `SoLoudCapture.instance`. 

Compare this to when you have `SoLoud().initCapture()`, you try to change it to `SoLoudCapture()`, but that doesn't exist at all.